lib/wanda/execution/evaluation.ex

```diff
@@ -119,8 +119,12 @@ defmodule Wanda.Execution.Evaluation do
        ) do
     expectation_results =
       agents_check_results
-      |> Enum.flat_map(fn %AgentCheckResult{expectation_evaluations: expectation_evaluations} ->
-        expectation_evaluations
+      |> Enum.flat_map(fn
```

Comment:
imho being explicit here is better than being implicit, I would `Enum.filter` `AgentCheckError` and then keep the flat map as it was

Reply:
:+1: done